src/app/components/Item/ItemFilters.jsx

```diff
@@ -29,25 +29,17 @@ const ItemFilters = ({ items, hasFilterApplied, numOfFilteredItems }, { router }
   const mapFilterIdsToLabel = {};
   itemFilters.forEach((filter) => {
     const optionsObjEntry = options[filter.type];
-    const filterOptions = optionsObjEntry || filter.options(items);
+    const filterOptions = optionsObjEntry || items.map(item => filter.options(item));
     if (!optionsObjEntry) options[filter.type] = filterOptions;
     filterOptions.forEach((option) => {
       mapFilterIdsToLabel[option.id] = option.label;
     });
   });
 
-  // join filter selections and add single quotes
-  const parsedFilterSelections = () => {
-    let filterSelections = itemFilters.map(
-      (filter) => {
-        const selection = query[filter.type];
-        if (Array.isArray(selection)) return selection.map(id => mapFilterIdsToLabel[id]);
-        return mapFilterIdsToLabel[selection];
-      }).filter(selections => selections);
-    if (filterSelections.flat) filterSelections = filterSelections.flat();
-    const joinedText = filterSelections.join("', '");
-    return `'${joinedText}'`;
-  };
+  const parsedFilterSelections = () => itemFilters
+    .map(filter => query[filter.type] && `${filter.type}: ${query[filter.type]}`)
+    .filter(selected => selected)
```

Comment:
Ellen's design does not have the filter type in the text. Not sure how big of a deal that is

Reply:
Personally I think the filter type is useful, I'd prefer to show this version and change it back if people don't like it.